docs/content/concepts/scenarios/README.md

```diff
@@ -0,0 +1,13 @@
+# Document the KCP Workspace objects
+
+Document key KCP objects, with they Yaml, that are created and modified during the key steps of MC Syncer deployment:
+
+1. Creation of a KCP Workspace (`k ws create`)

```

Comment:
```suggestion
1. Creation of a KCP Workspace (`kubectl ws create`)
```

nit: prefer not using aliases here - `k` does nothing on my system  and readers may be unfamiliar